spec/models/image_results_post_processor_spec.rb

```diff
@@ -34,7 +37,7 @@
       let(:total_results) { nil }
       let(:results) { nil }
 
-      it 'has a alt text, link URL, and thumbnaul URL' do
+      it 'has a alt text, link URL, and thumbnail URL' do
         expect(normalized_results[:results]).to be_nil
```

Comment:
This assertion doesn't seem to match the test description?

Reply:
Updated the description to match the test, please review